transit_model_procmacro/Cargo.toml

```diff
@@ -5,10 +5,22 @@ authors = ["Guillaume Pinot <texitoi@texitoi.eu>"]
 license = "AGPL-3.0"
 description = "A private procmacro crate for the transit_model crate"
 edition = "2018"
+autotests = false
 
 [lib]
 proc-macro = true
 
 [dependencies]
 syn = "0.11.11"
 quote = "0.3.15"
+
+[dev-dependencies]
+pretty_assertions = "0.6"
+trybuild = "1"
+transit_model = { path = "../" }
+transit_model_collection = { path = "../collection" }
+transit_model_relations = { path = "../relations" }
```

Comment:
it's very painfull to release cyclic dependencies

Reply:
Even for `dev-dependencies`?

Reply:
yes

Reply:
OK, so then, what do we do?

Reply:
Will [this change in `cargo@1.40.0`](https://github.com/rust-lang/cargo/pull/7333) help with the problem?

Reply:
according to the unit test yes